install/kubernetes/Makefile.digests.empty

```diff
@@ -0,0 +1,12 @@
+# File generated by .github/workflows/images-legacy-releases.yaml; DO NOT EDIT.
```

Comment:
This file is not generated by `.github/workflows/images-legacy-releases.yaml`. I would suggest to run a sed command against `Makefile.digests`, something like:
`sed 's/DIGEST\ :=\ ".*/DIGEST\ :=\ ""/g'`

Reply:
I thought about the sed and it just seemed more complicated than copying a template file over the top :sweat_smile: